manifests/init.pp

```diff
@@ -71,7 +71,7 @@
 # $ssl::                          Enable and set require_ssl in Foreman settings (note: requires passenger, SSL does not apply to kickstarts)
 #
 # $repo::                         This can be a specific version or nightly
-#
+
```

Comment:
this change should be reverted.

Reply:
gah, missed that on the rebase. fixed now.